src/components/Section/ProposalResultSection.tsx

```diff
@@ -51,8 +53,11 @@ export default React.memo(function ProposalResultSection({ proposal, loading, di
   ])}>
     <div className="DetailsSection__Content">
       <Loader active={loading} />
-      <div>
+      <div className="DetailsSection__Flex_Header_Labels">
         <Header sub>{l('page.proposal_detail.result_label')}</Header>
+        {Object.keys(votes || {}).length > 0 && <Anchor style={{ marginBottom: '5px', textTransform: 'uppercase' }} onClick={onOpenVotesList} >
```

Comment:
- cache the result of `Object.keys(votes || {}).length` using a `useMemo`
- use `font-size: 13px`
- move the css into the css file